site/content/en/docs/contributing/development-environment.md

```diff
@@ -205,10 +205,112 @@ description: 'Installing a development environment for different operating syste
 - If you choose to run CVAT in localhost: Select `server: chrome` configuration and run it (F5) to open CVAT in Chrome
 - Alternative: If you changed CVAT_UI_HOST just enter ```<YOUR_HOST_IP>:3000``` in your browser.
 
-
 You have done! Now it is possible to insert breakpoints and debug server and client of the tool.
 Instructions for running tests locally are available [here](/site/content/en/docs/contributing/running-tests.md).
 
+## Run Analytics
+To use `Analytics` in the development environment, you must have the Clickhouse, Vector and Grafana containers running.
+You can create a `docker-compose.debug.yml` with the following content and run all services
+with `docker-compose -f docker-compose.debug.yml up -d` command. Grafana will be available at `http://localhost:3001`.
```

Comment:
Why do you think it is a bad idea to put docker-compose.debug.yml into the repository? I'm not sure why you excluded cvat_db_debug and opa_db_debug services?